lib/vfscore/Config.uk

```diff
@@ -25,6 +25,16 @@ help
 	be changed with the following library parameters:
 	'vfs.rootfs', 'vfs.rootdev', 'vfs.rootflags', and 'vfs.rootopts'
 
+config NONLARGEFILE_SYSCALLS
+	bool "Provide proper legacy non largefile syscalls"
+	default y if LIBSYSCALL_SHIM_HANDLER
+	default n
+	help
+		Add different dirent and dirent64 structure and their respective
+		syscalls (i.e. treat Linux non largefile legacy syscalls properly).
+		If this option is not set, the dirent structure will be aliased to
+		dirent64, so will all other f/f64 functions.
```

Comment:
I'd mention that only the 64-bit variants of system calls are available via the syscall_shim, in case the option is off. Something like:
```diff
-dirent64, so will all other f/f64 functions.
+dirent64, so will all other f/f64 functions. If lib/syscall_shim
+is enabled, only the 64-bit version of system calls are registered.
```

Reply:
What does `f/f64` exactly mean? I can guess it.

Reply:
> What does `f/f64` exactly mean? I can guess it.

The non-largefile/largefile function pairs, I rephrased it, it was unclear.